src/zenml/utils/source_utils.py

```diff
@@ -337,14 +338,14 @@ def get_source_type(module: ModuleType) -> SourceType:
     if is_internal_module(module_name=module.__name__):
         return SourceType.INTERNAL
 
-    if is_standard_lib_file(file_path=file_path):
-        return SourceType.BUILTIN
-
     if is_distribution_package_file(
         file_path=file_path, module_name=module.__name__
     ):
         return SourceType.DISTRIBUTION_PACKAGE
 
+    if is_standard_lib_file(file_path=file_path):
```

Comment:
I think that it is better to fix the function itself not to false alert on non-builtins, rather doing this reorder.